pkg/maps/ctmap/per_cluster_ctmap_privileged_test.go

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Feel free to rename this to `per_cluster_ctmap_test.go`. With the removal of the `privileged_tests` build tag, privileged tests no longer need to be split into separate files, and the filename is getting rather long.